BossMod/Components/Knockback.cs

```diff
@@ -10,8 +10,10 @@ public abstract class Knockback : CastCounter
         public enum Kind
         {
             AwayFromOrigin, // standard knockback - specific distance along ray from origin to target
+            TowardsOrigin, // standard pull - "knockback" to source - forward along source's direction + 180 degrees
```

Comment:
I feel that properly supporting TowardsOrigin requires a few more changes - primarily min distance - for correctness.
Typically 'attract' effects cap the traveled distance, so that distance between target and origin is never less than a constant >= 0 (and maybe + source/target hitbox radiuses?) In any case, it can't become less than 0 - it would be an 'overshoot', which can't happen with attracts.

Reply:
yes, but i needed this kind towards origin for my sugaar module

Reply:
i manually made the calculations there for distance to boss and actual pull distance